Update customizing-in-experience-text-chat.md (#447)
Roblox/creator-docs: content/en-us/chat/customizing-in-experience-text-chat.md

TextChatService.OnIncomingMessage = onIncomingChatMessage
```

> ## Adding A System Message

Reviewer: ```suggestion
### Displaying System Messages
```

> By using [:DisplaySystemMessage](https://create.roblox.com/docs/reference/engine/classes/TextChannel#DisplaySystemMessage) you are able to create a system-like message. This can be used to greet users, or alert them when an ingame event is happening.

Reviewer: ```suggestion
By calling `Class.TextChannel:DisplaySystemMessage()`, you can display a system-like message, useful for greeting users or alerting them when an in-experience event is happening.
```

Reviewer: For future reference on linking within docs site, see:
https://github.com/Roblox/creator-docs/blob/main/STYLE.md#links

```lua title='LocalScript'

> game.TextChatService.TextChannels.RBXGeneral:DisplaySystemMessage("[Server] Hello There, " .. game.Players.LocalPlayer.Name)
> ```

Reviewer: ```suggestion
local Players = game:GetService("Players")
local TextChatService = game:GetService("TextChatService")

local generalChannel = TextChatService.TextChannels.RBXGeneral

generalChannel:DisplaySystemMessage("[Server] Hello " .. Players.LocalPlayer.Name)
```

Reviewer: Pulling out services to top, as per documentation general standards.

> ![image](https://github.com/Roblox/creator-docs/assets/74430503/41e01981-d308-4ab3-85ca-2999f466859c)

Reviewer: ```suggestion
```

Reviewer: I'd rather omit the image here. Users can easily run the code snippet to see how these system-like messages look on-screen.

> 
> 

Reviewer: ```suggestion

```
